### PR TITLE
World: Add functions for Kano World interactions

### PR DESCRIPTION
**Pull request by tombettany**
KanoComputing/engage#84
Add functions for posting and receiving share comments, liking and unliking shares,
following and unfollowing users.

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
+  * Add Kano World interraction functions
+
+ -- Team Kano <dev@kano.me>  Fri, 8 Oct 2015 17:24:00 +0100
+
 kano-profile (2.2.0-1) unstable; urgency=low
 
   * Tracker: Adding ping event
 
  -- Team Kano <dev@kano.me>  Fri, 25 Sep 2015 17:57:00 +0100
-
```
Comment (pazdera) [inline debian/changelog]: This doesn't look right

Comment (convolu) [inline debian/changelog]: Was this intentional? Does it make a difference?

**Comment by convolu** [comment]
Apart from a minor comment on the changelog, this looks ok.

File: debian/changelog (hunk 1 above)
Comment (tombettany) [inline debian/changelog]: I don't know what happened there but I'll fix it

**Comment by tombettany** [comment]
@pazdera @convolu Fixed the `debian/changelog` issue.

**Comment by convolu** [comment]
:shipit:

**Comment by pazdera** [comment]
:shipit:

**Pull request merged**